doc/source/whatsnew/v1.1.0.rst

```diff
@@ -99,7 +99,7 @@ Bug fixes
 Categorical
 ^^^^^^^^^^^
 
--
+- Bug in :class:`Categorical` that would ignore or crash when calling :meth:`Series.replace` with a list-like ``to_replace`` (:issue:`31720`)
```

Comment:
The issue is tagged with v1.0.2 milestone, I think the should go in that whatsnew file

Reply:
fixed